MangoAPI.Domain/ChatGPTOptions.cs

```diff
@@ -0,0 +1,12 @@
+namespace MangoAPI.Domain;
+
+public sealed class ChatGPTOptions
```

Comment:
If here all properties are `init` only, then can this class be converted to record with primary constructor?